doc/user/content/sql/create-sink.md

```diff
@@ -71,6 +71,7 @@ Field | Value | Description
 `ssl_certificate_location` | `text` | The absolute path to your SSL certificate. Required for SSL client authentication.
 `ssl_key_location` | `text` | The absolute path to your SSL certificate's key. Required for SSL client authentication.
 `ssl_key_password` | `text` | Your SSL key's password, if any.
+`ssl_key_password_env` | `text` | Use the value stored in the named environment variable as the value for `ssl_key_password`. Provides a more secure alternative to the `ssl_key_password` option.
```

Comment:
Do you mind if we drop the "secure" bit? Perhaps a bit pedantic, I don't want people to get the impression that storing passwords in the catalog is inherently insecure. I think instead we just say we don't write to disk, and let the users figure out if that is important to them.

So I guess here are some thoughts that I'd love if you wordsmith appropriately:

  * "Avoids storing the password on-disk in Materialize's catalog."
  * "Note that you while a source using this option is present in the catalog, you will not be able to start Materialize without providing a value for the specified environment variable."